src/virt.c

```diff
@@ -1960,12 +1971,15 @@ static int refresh_lists(struct lv_read_instance *inst) {
       for (int j = 0; j < xml_interfaces->nodeNr; ++j) {
         char *path = NULL;
         char *address = NULL;
+        char *type = NULL;
         xmlNodePtr xml_interface;
 
         xml_interface = xml_interfaces->nodeTab[j];
         if (!xml_interface)
           continue;
 
+        type = (char *)xmlGetProp(xml_interface, (const xmlChar *)"type");
```

Comment:
Do you need to check for error conditions here?

Reply:
No, but it does not an error check elsewhere.  Will fix.